### PR TITLE
refactor:Strip billing function to reosurce-controller

### PR DESCRIPTION
**Pull request by xiao-jay**
1、Strip billing function to reosurce-controller
2、remove infra calculate code from metering-controller
3、move resource and extensionResourcePrice CRD to controllers/common/metering/

### CODE REVIEW

**Comment by zzjin** [comment]
copilot:all

File: go.work [hunk 1]
```diff
@@ -16,4 +16,5 @@ use (
 	./staging/src/github.com/labring/image-cri-shim
 	./staging/src/github.com/labring/lvscare
 	./webhooks/whitelist
+	controllers/common/metering
```
Comment (zzjin) [inline go.work:19]:
```suggestion
	./controllers/common/metering
```

Comment (xiao-jay) [reply go.work]: done

File: controllers/account/config/crd/bases/account.sealos.io_accountbalances.yaml [hunk 2]
```diff
@@ -46,7 +46,32 @@ spec:
                 type: string
               owner:
                 type: string
+<<<<<<< HEAD
```
Comment (zzjin) [inline controllers/account/config/crd/bases/account.sealos.io_accountbalances.yaml:49]: Wrong commit.

Comment (xiao-jay) [reply controllers/account/config/crd/bases/account.sealos.io_accountbalances.yaml]: done

**Comment by xiao-jay** [comment]
make install correct
```
➜  metering git:(refector-metering) ✗ make install
/Users/jie/Desktop/golang/sealos/controllers/common/metering/bin/controller-gen rbac:roleName=manager-role crd paths="./..." output:crd:artifacts:config=config/crd/bases
test -s /Users/jie/Desktop/golang/sealos/controllers/common/metering/bin/kustomize || { curl -Ss "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" --output install_kustomize.sh && bash install_kustomize.sh 4.2.0 /Users/jie/Desktop/golang/sealos/controllers/common/metering/bin; rm install_kustomize.sh; }
/Users/jie/Desktop/golang/sealos/controllers/common/metering/bin/kustomize build config/crd | kubectl apply -f -
customresourcedefinition.apiextensions.k8s.io/extensionresourceprices.metering.common.sealos.io unchanged
customresourcedefinition.apiextensions.k8s.io/resources.metering.common.sealos.io unchanged
```

e2e test correct 
https://github.com/xiao-jay/sealos/actions/runs/4541385172/jobs/8003522485

**Review by zzjin — APPROVED** [review]

**Pull request merged**